zookeeper-api/src/main/java/org/apache/helix/zookeeper/api/factory/RealmAwareZkClientFactory.java

```diff
@@ -19,5 +19,13 @@
  * under the License.
  */
 
+import org.apache.helix.zookeeper.api.client.RealmAwareZkClient;
+
+
+/**
+ * Creates an instance of RealmAwareZkClient.
+ */
 public interface RealmAwareZkClientFactory {
+  RealmAwareZkClient buildZkClient(RealmAwareZkClient.RealmAwareZkConnectionConfig connectionConfig,
```

Comment:
To achieve feature parity of HelixZkClientInterface, shall we add this constructor too?

`   public RealmAwareZkClient buildZkClient(
      RealmAwareZkClient.RealmAwareZkConnectionConfig connectionConfig) {
     return buildZkClient(connectConfig, new RealmAwareZkClient.RealmAwareZkClientConfig());
   }`

Reply:
Sure, that's a good idea. Let me add it as well.